app/server/lib/NSandbox.ts

```diff
@@ -609,11 +619,20 @@ function sandboxed(options: ISandboxOptions): SandboxProcess {
  * been installed globally.
  */
 function unsandboxed(options: ISandboxOptions): SandboxProcess {
-  const {args: pythonArgs, importDir} = options;
+  const {args, importDir} = options;
   const paths = getAbsolutePaths(options);
+
+  const pythonArgs = [...args];
   if (options.useGristEntrypoint !== false) {
-    pythonArgs.unshift(paths.main);
+    pythonArgs.push(paths.main);
+  }
+  if (options.comment) {
```

Comment:
Why is this an `unsandboxed` thing now?

Reply:
Previously `NSandboxCreator` decided this should be in the command's argument list. 

However, not all commands are happy with that - specifically, `node` wasn't happy getting a comment forced into the start of its argument list (when Pyodide uses `spawn`)

This makes the individual sandbox spawner decided how the comment should be integrated into the process' command line, rather than leaving it up to `NSandboxCreator`, so that different sandboxes can integrate it in a way that works for them.